src/components/pages/404-page/NotFoundPage.styles.ts

```diff
@@ -0,0 +1,55 @@
+import { SxProps } from '@mui/material/styles';
+import { Theme } from '@mui/system';
+
+export const pageLayout: SxProps<Theme> = {
+  width: '100%',
+  display: 'flex',
+  minHeight: '100vh',
+  flexDirection: 'column',
+  alignItems: 'center',
+  backgroundImage: `url("/icons/404-page/leaves-left.svg"),
+                     url("/icons/404-page/leaves-right.svg"),
+                     url("/icons/404-page/clouds-desktop.svg")`,
+  backgroundRepeat: 'no-repeat, no-repeat, no-repeat',
+  backgroundPosition: 'left bottom, right bottom, center center',
+  backgroundSize: '29%, 29%, 100%',
+};
+
+export const notFoundText: SxProps<Theme> = {
+  width: { desktop: '353px', mobile: '226px' },
+  height: { desktop: '62px', mobile: '54px' },
+  textAlign: 'center',
+  marginTop: { desktop: '64px', mobile: '63px' },
+  marginBottom: { desktop: '57px', mobile: '44px' },
+  typography: { desktop: 'h5', mobile: 'body2' },
```

Comment:
you need replace all typography as 
import theme from '@/styles/theme';
`typography: { desktop: theme.typography.h5, mobile:  ...},`